src/screens/OtherProfile/elements/BioAndChat.js

```diff
@@ -0,0 +1,190 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {Pressable, StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+
+import TextAreaChat from '../../../components/TextAreaChat';
+import ToggleSwitch from '../../../components/ToggleSwitch';
+import useSaveAnonChatHook from '../../../database/hooks/useSaveAnonChatHook';
+import {ANON_PM, SIGNED} from '../../../hooks/core/constant';
+import {colors} from '../../../utils/colors';
+import {fonts} from '../../../utils/fonts';
+import {sendAnonymousDMOtherProfile, sendSignedDMOtherProfile} from '../../../service/chat';
+import {trimString} from '../../../utils/string/TrimString';
+
+const CHANNEL_BLOCKED = 'Channel is blocked';
+
+const BioAndChat = (props) => {
+  const {
+    isAnonimity,
+    bio,
+    openBio,
+    dataMain,
+    isSignedMessageEnabled,
+    showSignedMessageDisableToast,
+    loadingGenerateAnon,
+    avatarUrl,
+    anonProfile,
+    username,
+    toggleSwitch,
+    isAnonimityEnabled
+  } = props;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The destructuring of props can be simplified by using default props. This will make the code cleaner and easier to read.
```diff
- const {
-     isAnonimity,
-     bio,
-     openBio,
-     dataMain,
-     isSignedMessageEnabled,
-     showSignedMessageDisableToast,
-     loadingGenerateAnon,
-     avatarUrl,
-     anonProfile,
-     username,
-     toggleSwitch,
-     isAnonimityEnabled
-   } = props;
+ const {
+     isAnonimity = false,
+     bio = '',
+     openBio = () => {},
+     dataMain,
+     isSignedMessageEnabled = false,
+     showSignedMessageDisableToast = () => {},
+     loadingGenerateAnon = false,
+     avatarUrl = '',
+     anonProfile,
+     username,
+     toggleSwitch,
+     isAnonimityEnabled = false
+   } = props;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->